### DISCUSSION TITLE
修改功能

### DISCUSSION THREAD

**Discussion by 16rab** [nano71/GlutAssistantN#7] [Ideas]
课程修改功能（添加、删除、移动、编辑）

**Comment by nano71** [comment]
添加：可以考虑
其他：会导致新旧课表冲突，除非你有合适的理由

**Comment by 16rab** [comment]
因为获取课表只有一次 后续更新也不会频繁 所以冲突就直接覆盖好了 主要我想把他当做一个日程表使用 记录到梦空间日程应该会蛮方便的
